DbTables/inc/CalCosmicT0Align.hh

```diff
@@ -0,0 +1,70 @@
+#ifndef DbTables_CalCosmicT0Align_hh
+#define DbTables_CalCosmicT0Align_hh
+
+
+#include <string>
+#include <iomanip>
+#include <sstream>
+#include <map>
+#include "Offline/DbTables/inc/DbTable.hh"
+#include "Offline/DataProducts/inc/CaloSiPMId.hh"
+
+namespace mu2e {
+
+  class CalCosmicT0Align : public DbTable {
+  public:
+
+    class Row {
+    public:
+      Row(CaloSiPMId  roid, float Tcorr, float Terr, float chisq):_roid(roid),_Tcorr(Tcorr), _Terr(Terr), _chisq(chisq){}
+      CaloSiPMId   roid() const { return _roid;}
+      float Tcorr() const { return _Tcorr; }
+      float Terr() const { return _Terr; }
+      float chisq() const { return _chisq; }
+
+    private:
+      CaloSiPMId   _roid;
+      float _Tcorr;
+      float _Terr;
+      float _chisq;
+    };
+
+    constexpr static const char* cxname = "CalCosmicT0Align";
+
+    CalCosmicT0Align():DbTable(cxname,"cal.cosmicT0align","roid,Tcorr,Terr,chisq"){}
```

Comment:
In the quotes here is the database table name and column names, which are case insensitive, please make these "T" into "t".  We will want to make this change in existing tables, but that can wait for another PR.